app/guid-node/registrations/styles.scss

```diff
@@ -1,4 +1,5 @@
 // stylelint-disable max-nesting-depth, selector-max-compound-selectors
+@import 'app/styles/components';
```

Comment:
Tried replacing the instances of `@import` with `@use`, and it seems like there may be some problems with how sass and ember-css-modules interacts. It builds, but I'm getting an error like so

```
Sass Syntax Error (SassCompiler) in /var/folders/81/p7fqb8j5043_32pnb84m8n4h0000gp/T/broccoli-77582lW6C1CrcSZzY/out-1045-broccoli_merge_trees//addon.scss:7043:1

Error: @use rules must be written before any other rules.
     ╷
7043 │ @use 'app/styles/components';
     │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     ╵
  /var/folders/81/p7fqb8j5043_32pnb84m8n4h0000gp/T/broccoli-77582lW6C1CrcSZzY/out-1045-broccoli_merge_trees/addon.scss 7043:1  root stylesheet
```

Not sure if this is an easy fix or not, but I could make an analysis ticket for this issue. Might be best to change all the `@import`s in one fell swoop too.

Reply:
oh i see... yeah judging by [this old issue](https://github.com/dfreeman/ember-css-modules-sass/issues/3) `@use` probably won't work nicely with ember-css-modules-sass (which hasn't been substantially updated since before [sass's new module system came to be](https://github.com/sass/sass/blob/main/accepted/module-system.md#timeline))

so yeah, probably best to stick to how it's done and plan to someday consider optimizing the 20k+ lines of css currently in ember-osf-web dist...

Reply:
Ahh, great detective work! Yea, css-modules and sass are quite nice and I enjoy using them, but it's definitely something to consider reworking for longevity. Although I foresee us punting this further until it becomes a critical and unavoidable problem